src/api/guardian_filter.js

```diff
@@ -1 +1,13 @@
-module.exports = response => {};
+module.exports = response => {
+  let results = response.results.reduce((articles, article) => {
+    articles.push({
+      id: article.id,
+      image: article.fields.thumbnail,
+      headline: article.fields.headline,
+      trailText: article.fields.trailText,
+      bodyText: article.fields.bodyText
```

Comment:
also for those now you just do `fields.x` ( x is whatever )

Reply:
done